README.md

```diff
@@ -48,7 +53,14 @@ The QF Network executes smart contracts in the PolkaVM virtual machine and requi
 
 ## Deploy and call a smart contract
 
-See [guides.md](guides.md).
+```bash
+./build_polkavm.sh increment-counter
+
+cd cli
+npx ts-node uploadAndExecute.ts wss://test.qfnetwork.xyz ../output/increment-counter.polkavm
```

Comment:
Looks like this doesn't work in TestNet because you use Alice's seed in `cli/uploadAndExecute.ts ` and Alice doesn't have tokens.

Reply:
Yes, I've tested locally. I think I'll add a link to guide for running local node

Reply:
Done